internal/operators/cnv/cnv_operator.go

```diff
@@ -68,7 +68,7 @@ func (o *operator) GetDependencies(cluster *common.Cluster) ([]string, error) {
 		return []string{}, err
 	}
 
-	minOCPVersionForLVM, err := version.NewVersion(lvm.LvmMinOpenshiftVersion)
+	minOCPVersionForLVM, err := version.NewVersion(lvm.LvmoMinOpenshiftVersion)
```

Comment:
```suggestion
	minOCPVersionForLVMS, err := version.NewVersion(lvm.LvmsMinOpenshiftVersion)
```

Reply:
in CNV - we want to install LVMS only when OCP version >= 4.12, in 4.11 we use LSO

Reply:
got it